### ISSUE TITLE
Malware present in build artifact

### ISSUE THREAD

**Issue by bryphe** [appveyor/ci#1913]
Hi,

One of my users reported malware detected in a built artifact for my app: https://github.com/onivim/oni/issues/889

The infected file is:
- `C:\Program Files (x86)\Oni\resources\app\node_modules\vscode-ripgrep\bin\rg.exe`

With the threat described as:
https://www.sophos.com/en-us/threat-center/threat-analyses/viruses-and-spyware/Mal~EncPk-ZC/detailed-analysis.aspx

It appears that, when running `npm install vscode-ripgrep` locally, the binary is clean - but when taken from the AppVeyor CI servers, it is infected.

I've created a minimal repo here to demonstrate the issue:
https://github.com/bryphe/test-ripgrep-appveyor-build

And the infected artifact for inspection is here:
https://ci.appveyor.com/project/bryphe/test-ripgrep-appveyor-build/build/job/y1ba0ykhlntw5pdj/artifacts

Note that, if I run `npm install vscode-ripgrep` locally, the binary is clean - but when I produce the artifact via the `AppVeyor CI` servers, it is infected.

A few other interesting points that may help:
- `x64` and `x86` binaries are both impacted
- This occurred both with `os: unstable` and without setting `os`
- My app has multiple executables (including `Oni.exe` and `nvim.exe`) but only `rg.exe` is impacted.

**Comment by FeodorFitsner** [comment]
Indeed, I've I noticed that AppVeyor build downloads `ripgrep-0.6.0-patch.0-win32-ia32.zip`. Both `rg.exe` files from that archive and downloaded from build artifacts are identical and has the same SHA1 `ef1543351562958240cca9da70db2d1f52ffa114`.

Btw, I'm on the latest Windows 10 with up-to-date Defender database and it detects nothing suspicious in that `rg.exe` file.

**Comment by wodin** [comment]
FYI - https://www.virustotal.com/#/file-analysis/MjU0ZDUwM2U1M2Q0YTk5Mjc2MGE3NTViNzEzZjRlODU6MTUxMDU2MTMyOQ==

Cylance - Unsafe
Sophos AV - Mal/EncPk-ZC

All others detect no problems.
